portals/publisher/src/main/webapp/source/src/app/data/AuthManager.js

```diff
@@ -204,33 +204,43 @@ class AuthManager {
      * @param {*} api
      */
     static isRestricted(scopesAllowedToEdit, api = {}) {
+        const user = AuthManager.getUser();
+
+        if (!user) {
```

Comment:
Shall we pair this if statement with the one below? Multiple if statements would be unnecessary.